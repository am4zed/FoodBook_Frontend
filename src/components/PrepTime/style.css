.recipe-prep-time {
    grid-area: prepTime;
    margin: 0;
  }