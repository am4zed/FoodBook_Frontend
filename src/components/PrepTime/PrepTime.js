import React from "react";
import "./style.css"

const PrepTime = props => <h4 className="recipe-prep-time">Preparation time : {props.totalTime}</h4>;

export default PrepTime;
