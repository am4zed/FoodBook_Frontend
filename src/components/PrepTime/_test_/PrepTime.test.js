import React from 'react';
import PrepTime from '../PrepTime';
import { arrowFunctionExpression } from '@babel/types';

describe("PrepTime", () => {


 it("Renders without crashing", () => {
   shallow(<PrepTime/>)
 });

 it("Has an <h4> element", () => {
  const wrapper = shallow(<PrepTime/>)
  expect(wrapper.exists("h4")).toBeTruthy();
 })

 it("snapshot", () => {
   const wrapper = shallow(<PrepTime totalTime={10}/>)

   expect(wrapper).toMatchSnapshot();
 })
})
