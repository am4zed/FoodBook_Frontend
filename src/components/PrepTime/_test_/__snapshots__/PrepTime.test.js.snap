// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrepTime snapshot 1`] = `
<h4>
  Preparation time : 
  10
</h4>
`;

exports[`totalTime snapshot 1`] = `
<h4>
  Preparation time : 
  10
</h4>
`;
